recipes/strling/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cp "${CC}" "${BUILD_PREFIX}/bin/gcc"
+nimble install -y --verbose
+chmod a+x strling
```

Comment:
This line can also be removed (the created binary is already executable)

Reply:
right